modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -129,6 +130,11 @@ $menu-v-space: 25px;
 $admin-bar-h-pad: 20px;
 $admin-bar-border: 1px solid var(--a-base-9);
 
+.apos-admin-bar-wrapper {
+  z-index: $z-index-manager-toolbar;
```

Comment:
Do you still need z-index on ` .apos-admin-bar`?

Reply:
good q, will check before merging